services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/paatos/PaatosEntity.kt

```diff
@@ -0,0 +1,50 @@
+package fi.hel.haitaton.hanke.paatos
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.EnumType
+import jakarta.persistence.Enumerated
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import java.time.LocalDate
+import java.util.UUID
+
+enum class PaatosTyyppi {
```

Comment:
Would it be better to have these enums in Paatos.kt instead of entity class file? Semantically speaking.

Reply:
I don't see a semantical difference whether they're in Paatos or PaatosEntity.

I moved the enums to a separate file.

Reply:
Okay. For me it seems a more clear choice to have domain thingies outside of entity declarations.